addons/web_editor/static/src/js/wysiwyg/widgets/chatgpt_alternatives_dialog.js

```diff
@@ -50,48 +52,59 @@ export class ChatGPTAlternativesDialog extends ChatGPTDialog {
 
     switchAlternativesMode(ev) {
         this.state.alternativesMode = ev.currentTarget.getAttribute('data-mode');
-        this.state.messages = [];
-        this._generateAlternatives();
+        this._generateAlternatives(1);
     }
 
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
 
-    async _generateAlternatives() {
+    async _generateAlternatives(numberOfAlternatives) {
```

Comment:
Having both `this.props.numerOfAlternatives` AND `numberOfAlternatives` makes it confusing IMO, unless you do:
```js
async _generateAlternatives(numberOfAlternatives = this.props.numberOfAlternatives) {
```

Reply:
Done.